SYCL/CommandGraph/graph_finalize_empty.cpp

```diff
@@ -18,8 +18,8 @@ int main() {
       graph;
   auto graphExec = graph.finalize(testQueue.get_context());
 
-  testQueue.submit(graphExec);
+  testQueue.submit([&](handler &h) { h.exec_graph(graphExec); });
```

Comment:
This is very nitpicky but in other uses in these tests we use `cgh` for the handler variable.